lib/components/Modal.js

```diff
@@ -7,12 +7,15 @@ var elementClass = require('element-class');
 var renderSubtreeIntoContainer = require("react-dom").unstable_renderSubtreeIntoContainer;
 
 var SafeHTMLElement = ExecutionEnvironment.canUseDOM ? window.HTMLElement : {};
+var AppElement = document.body;
```

Comment:
Sorry, I don't understand. If AppElement === {appendChild: function() {}}, surely AppElement.appendChild(this.node) will do nothing. So the modal will not be appended to the DOM.